docs/Resources-Appendices/operators.md

```diff
@@ -39,22 +39,27 @@ z = x / ++x;
 Programs should not depend upon the order of evaluation of expressions, except as guaranteed by ANSI C for the following operators:
 
 ```
- 1.  a, b           comma operator (not the comma between arguments) 
+ 1.  a, b           comma operator (not the comma between arguments)
  2.  a && b         logical and
  3.  a || b         logical or
- 4.  a ? b : c      conditional 
+ 4.  a ? b : c      conditional
 ```
 
 All of these guarantee that expression `a` will be computed before expression `b` (or `c`).
 
 In addition, when a function-call takes place all arguments are evaluated before control transfers to the function.
 
-```
-5.  a(b)           function call 
-```
+:::note 
 
-ANSI C++ guarantees that each full expression will be evaluted before going on.
+ 5.  a(b) function call
+ 
+:::
 
-```
-6.  each full expression 
-```
+
+ANSI C++ guarantees that each full expression will be evaluated before going on.
+
+:::note 
+
+6. each full expression
```

Comment:
In part 6, I suggest to put them into a code block similarly like before. It is better look in code block.